src/engines/aurora/camera.cpp

```diff
@@ -77,6 +78,8 @@ bool handleCameraKeyboardInput(const Events::Event &e) {
 	} else
 		return false;
 
+	//TODO: Move this somewhere else
```

Comment:
Then move this somewhere else? :P

(Also, there needs to be space after `//`)

Reply:
Because the player is currently not moving, i attached the listener to the camera. This should be changed of course, when the player can move.

Reply:
Then please state something of this effect in the comment, so that everybody knows what this is about :)